docs/dataset/advanced-usage/quality-standard-results.md

```diff
@@ -0,0 +1,42 @@
+---
+icon: kolena/take-action-16
+
+---
+
+# :kolena-take-action-20: Programatically Compare Models
+
+## Download Quality Standard Result
+
+The **Quality Standard** should contain the key performance metrics a team uses to evaluate a model's performance on a
+dataset.
+
+The SDK provides [a
+function](../../reference/experimental/index.md#kolena._experimental.quality_standard.download_quality_standard_result)
+to download a dataset's quality standard result. This enables users to automate processes surrounding a Quality
+Standard's result.
+
+The return value is a multi-index DataFrame with indices `(stratificaiton, test_case)` and columns `(model, eval_config,
+metric_group, metric)`.
+
+<figure markdown>
+![Quality Standard Result](../../assets/images/‎quality_standard_diagram_light.jpeg#only-light)
+![Quality Standard Result](../../assets/images/‎quality_standard_diagram_dark.jpeg#only-dark)
+<figcaption>Quality Standard Result</figcaption>
+</figure>
+
+## Examples
```

Comment:
Working on another example for custom reporting eg. automate a weekly email with bottom 10 performing test cases to boss

Reply:
Going to scratch this example for now